utils/S3UtilsMongoClient.js

```diff
@@ -342,18 +362,75 @@ class S3UtilsMongoClient extends MongoClientInterface {
                         });
                         return callback(err);
```

Comment:
when `callback` is called with error, we should not process the MPUs anymore (which would call `callback` eventually as well)